### DISCUSSION TITLE
What is this?

### DISCUSSION THREAD

**Discussion by Tangerino** [micropython/micropython#14664] [General]
![CleanShot 2024-05-20 at 10 33 40](https://github.com/micropython/micropython/assets/3164100/03dbc30b-7180-4ea9-ab15-13c248830ab5)

**Comment by dlech** [comment]
Spam :unamused:

**Comment by robert-hh** [comment]
Spam Waves, hard to suppress. Stopping it was easier in the old forum.

**Reply by Josverl** [reply]
Owners/maintainers can block, all can report and then GH will take down the account.
But it's a cat&mouse

**Reply by scruss** [reply]
and the process is made complicated for users: the amount of captchas and nonsense I have to go through to report one spammer is a million times the effort the spammer put into posting their annoying noise. And once you've reported a couple of spam account, github limits you from reporting more. So the spammers will always win.

**Reply by robert-hh** [reply]
The only fast response I have is to delete the discussion items.
